### ISSUE TITLE
gitlab does not send email - gitlab continer unhealthy status

### ISSUE THREAD

**Issue by mradeschnig** [sameersbn/docker-gitlab#2337]
I have the docker-gitlab image 13.10.2 installed on Ubuntu 18.04 (Docker Engine - Community 20.10.5).
First I have the problem that I could not test sending email which I solved with another incident. Sending
emails from the gitlab console works. 
But still I do not get emails for passwort recovery as well no welcome mail after registration is sent to the new user.

How to find out why the gitlab container is in unhealthy state and second whats the problem
that gitlab does not send emails.

Obviously I don't see anything wrong except maybe this line in the logs ( I use portainer so I guess this aggregates all logs together):
/sbin/entrypoint.sh: line 19:   666 Terminated              /usr/bin/supervisord -nc /etc/supervisor/supervisord.conf

Kind Regards

**Comment by kkimurak** [comment]
Sorry for not helpful about email, but current container always reports unhealthy if you're running with relative URL (I'm about to submit a PR later).  
And, I also sometimes see such logs even everything works fine.

**Comment by mradeschnig** [comment]
I use a relative url (/git ) than this might be the reason for the heathstatus = unhealthy.
Thank you.

**Comment by mradeschnig** [comment]
A friend helped me. He think i made it wrong to use the internal server name as GITLAB_HOST instead the external  url.
And in fact this was right. Email works now, but all the actions to pull/clone the repositories also have now the wrong url
and can not be reached from external. I use an Apache ReverseProxy to redirect the external call to the correct server.
So in fact what is the correct proceeding? Or do I need to change something for the ReverseProxy? In fact it is strange as the mail server is external.

**Comment by mradeschnig** [comment]
The error (misconfiguration) is solved. It was the cofig: I set the GITLAB_HOST= https://subdomain.mydomain.com
but it only works with GITLAB_HOST= subdomain.mydomain.com.
I realized it when I put the local hostname there.
I wonder that it is not checked during creation or protokol is cut off.
Some functions work, some like email won't work with the protokol included.
Maybe a check would make sense the it is a correct hostname.
